### PR TITLE
So Much Maths

### LINKED ISSUE #20: Wizard's hitbox allows you to float on ledges
### Contact Details

@MattsAttack

### What happened?

Basically, since the wizard's hand and hat sticks out, he can get stuck and it will keep you from falling.

### Reproduction steps

1. Try to jump on/off ledge

### What should have happened?

You should have fallen!

### Date

2024-02-01

### What device are you seeing the problem on?

_No response_

### OS

Windows

### Browser

_No response_

### Context

image

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

_**CC:** @lishaduck_

### LINKED ISSUE #24: Rocks don't Export Correctly
### Contact Details

@lishaduck

### What happened?

When exporting for web, the rocks don't display their labels correctly. Additionally, the numbers don't fire.

### Reproduction steps

1. Export for web
2. Follow [the instructions](https://docs.godotengine.org/en/stable/tutorials/export/exporting_for_web.html#serving-the-files:~:text=gzip%20precompression\)-,Tip,should%20use%20an%20established%20web%20server%20such%20as%20Apache%20or%20nginx.,-Calling%20JavaScript%20from) to preview the built app.
3. Look at the rocks.

Alternatively,

1. Click "Remote Debug," aka [One-Click Deploy](https://docs.godotengine.org/en/stable/tutorials/export/one-click_deploy.html)
2. Look at the rocks.

### What should have happened?

The rocks should be labeled and the numbers should fire.

### Date

2024-02-02

### What device are you seeing the problem on?

Macbook Air 2020 (M1)

### OS

macOS

### Browser

Chrome

### Context

I've verified that it all still works fine in debug for macOS.

1.  Export for Web
    Screenshot 2024-02-02 at 7 42 50 PM
2.  Debug for Web
    Screenshot 2024-02-02 at 7 46 56 PM

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by lishaduck**
## Description

This PR just rebases #27 onto main

---

## Type of Change

- ✨ New feature

## Checklist

> Before submitting the PR, please make sure you do the following

- [x] Read the Contributing Guidelines.
- [x] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
- [x] Fill out this template.
- [x] Log your hours.
- [ ] Check that commits follow the Angular commit convention, more or less.
- [ ] Ideally, include relevant tests that fail without this PR but pass with it (if applicable).

## Tested on

- macOS 14

### CODE REVIEW

**Comment by lishaduck** [comment]
@MattsAttack, added anti-math juice!
@iiAzui, finished rebase a while ago. Forgot to mention it.

**Comment by lishaduck** [comment]
FYI, @iiAzui, this also includes the changes from the maps branch.

File: project.godot [hunk 1]
```diff
@@ -11,10 +11,17 @@ config_version=5
...
 config/features=PackedStringArray("4.2", "GL Compatibility")
+boot_splash/bg_color=Color(0.141176, 0.141176, 0.141176, 1)
 config/icon="res://icon.svg"
 
+[display]
+
+window/size/viewport_width=1280
+window/size/viewport_height=800
+window/stretch/aspect="ignore"
+
```
Comment (lishaduck) [inline project.godot:24]: @iiAzui, what does this do?

File: menu/font.otf
Comment (lishaduck) [inline menu/font.otf:1]: @iiAzui, what font is this? We'll need to attribute it, and probably include the font's license here.

File: menu/menu.webp
Comment (lishaduck) [inline menu/menu.webp:1]: Could you convert this to a PNG for consistency, @iiAzui?

File: number_balls/number.tscn
Comment (lishaduck) [inline number_balls/number.tscn:1]: I'm not sure what these changes are for, but no matter.

File: rotation_point.gd [hunk 2]
```diff
@@ -29,7 +29,7 @@ func _process(_delta: float) -> void:
 
 
 func fire(num: int) -> void:
-	const WOWZA := preload("res://number_balls/Number.tscn")
+	const WOWZA := preload("res://number_balls/number.tscn")
```
Comment (lishaduck) [inline rotation_point.gd:32]: _ALWAYS_ lowercase file names, @iiAzui.

File: project.godot (hunk 1 above)
Comment (lishaduck) [reply project.godot:24]: I see. It sets the default viewport.

File: menu/menu.webp
Comment (iiAzui) [reply menu/menu.webp:1]: we can yes

File: menu/font.otf
Comment (iiAzui) [reply menu/font.otf:1]: https://www.fontspace.com/legendary-runes-font-f66171

**Review by iiAzui — APPROVED** [review]

**Pull request merged**